src/test/groovy/graphql/DataFetcherTest.groovy

```diff
@@ -98,7 +98,7 @@ class DataFetcherTest extends Specification {
         given:
         def environment = env(GraphQLString)
         when:
-        def result = new PropertyDataFetcher("publicField").get(environment)
+        def result = new FieldDataFetcher("publicField").get(environment)
         then:
         result == "publicValue"
     }
```

Comment:
Do we have tests for Boolean and boolean fields already (I dont have code in front of me)

If not can we add them

Also doesnt beanUtils handle 

     boolean getBooleanMethod
     Boolean getBooleanMethod
     boolean isBooleanMethod
     Boolean isBooleanMethod

Can we add tests for those please